file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -0,0 +1,70 @@
+INTERFACE zif_aff_ntty_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! $values { @link zif_aff_ntty_v1.data:co_profile }
+    "! $default { @link zif_aff_ntty_v1.data:co_profile.text }
+    ty_profile TYPE c LENGTH 20,
+
+    "! $minimum 0
+    "! $maximum 2147483647
+    ty_max_length TYPE i,
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Language Dependent</p>
+      "! Language dependent
+      "! $showAlways
+      language_dependency TYPE abap_bool,
+      "! <p class="shorttext">Multiple Notes</p>
+      "! Multiple notes possible
+      "! $showAlways
+      multiple            TYPE abap_bool,
+      "! <p class="shorttext">Hide Title</p>
+      "! Hides the title
+      no_subject          TYPE abap_bool,
+    END OF ty_general_information,
+
+    "! <p class="shorttext">Content Settings</p>
+    "! Content settings
+    BEGIN OF ty_content_settings,
+      "! <p class="shorttext">Maximum Length</p>
+      "! Maximum length
+      max_length TYPE ty_max_length,
+      "! <p class="shorttext">Formatting Profile</p>
+      "! Formatting profile
+      profile    TYPE ty_profile,
+    END OF ty_content_settings,
+
+    "! <p class="shorttext">Notetype</p>
+    "! Notetype
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version      TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header              TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
Does your object type support also `"keyUser"` as abap language version? If yes, this header is fine. Otherwise, it would be `ty_header_60_cloud`

Reply:
I changed it to `ty_header_60_cloud`. Thank you for the hint.